src/com/hms/flexyosisoftconnector/DataPoint.java

```diff
@@ -16,8 +16,16 @@
    public static final byte TYPE_INT     = 2;
    public static final byte TYPE_DWORD   = 3;
 
+   protected long epochTime;
    protected String timestamp;
 
+   public long getEpcohTime() {
+	   return epochTime;
+   }
+   
+   public void setEpochTime(long time) {
+	   epochTime = time;
+   }

```

Comment:
there should probably be a blank line after this method to keep things consistent